### ISSUE TITLE
Business Rule Question: Should local storage be cleared when updating the the editor?

### ISSUE THREAD

**Issue by ghorsey** [w8tcha/CKEditor-AutoSave-Plugin#113]
It's just something I've noticed.

After loading the content from local storage into the editor via either a diff or the autoLoad option, the code clears that local storage.  (It is then saved again after the delay because a changed event is fired).

It seems to me, that the local storage cache should only be cleared:
* if they cancel the diff window
* if they save the content
* if the the content aged out
* when the local storage cached content matches the editor content.

*shrug* Just something I noticed...

**Comment by ghorsey** [comment]
I ended up making two minor tweaks.  

1. On load don't clear the cache
2. On save delay hit, check if the current cache matches the content.  If yes do not resave; otherwise save

What I noticed after disabling the cache clear on load, another save notification would happen (because it triggers the change event).  I thought it was a weird experience after modification number 1 above.

*shug* 

Great work by the way!  You're component has really helped me out!
